examples/vortex-mpi.py

```diff
@@ -102,7 +102,13 @@ def main(actx_class, ctx_factory=cl.create_some_context, use_logmgr=True,
     else:
         actx = actx_class(comm, queue, allocator=alloc, force_device_scalars=True)
 
-    # timestepping control
+    if use_esdg and not actx.supports_nonscalar_broadcasting:
+        raise RuntimeError(
+            f"{actx} is not a suitable array context for using flux-differencing. "
+            "The underlying array context must be capable of performing basic "
+            "array broadcasting operations. Use PytatoPyOpenCLArrayContext instead."
+        )    # timestepping control
```

Comment:
How come only `vortex-mpi` has this? Is it different from the other examples?

Reply:
ESDG is lazy with overintegration ONLY.  It does not work without that, and will just error.  In the other examples it is set so that enabling ESDG automatically enables lazy and overintegration - so this would never be triggered.